src/components/LoginComponent/Login.js

```diff
@@ -0,0 +1,74 @@
+import {React, useState} from 'react';
+import Bar from '../BarComponent/Bar';
+
+const buttonStyle = {
+    borderRadius: '40px',
+    height: '20%',
+    width: '50%',
+    margin: '1vh',
+    fontFamily: 'Anonymous Pro',
+    fontSize: '3vw',
+    background: '#ADADAD',
+    border: 'none',
+  }
+
+function Login(){
```

Comment:
change file and function name to LoginPage() to have consistency across other pages